### ISSUE TITLE
Increase Eye size via script unity

### ISSUE THREAD

**Issue by ragavendranbala** [googlevr/cardboard#375]
Is there's any api available to increase the eye size of the app without scanning a QR code? Most of local market headsets doesn't have QR codes to scan. I want to target my app to view on 40-42mm lens. 
Kindly let me know to manually set the Eye size via script in unity.

**Comment by DaReign** [comment]
Hi Arilow ;)

This is what I get with legacy sdk without any configuration just fresh start
![image](https://github.com/googlevr/cardboard/assets/11806754/2db7fa1b-e2ed-4312-9243-31a137995c95)

This is application https://play.google.com/store/apps/details?id=com.Granddesign.VRRelaxWalk

And this is what I get with new sdk without any configuration just fresh start
![image](https://github.com/googlevr/cardboard/assets/11806754/b290ab03-30eb-4576-8813-90159d9fc10d)

This is application (you have to choose first environment from the list) https://play.google.com/store/apps/details?id=com.Granddesign.VrBestGameCardboardRelaxEnviromentApp

Screens are from Samsung Galaxy A53

Why eye size is smaller ? How can archive similar effect in new sdk ? Forcing user to do something is bad approach. Is there an option to render "full screen" for each eye ? Maybe I just need to change something in code so every user at start would have full render filling all available space ?

**Comment by DaReign** [comment]
@arilow Is there any possibility to make bigger eye using SaveDeviceParams by passing some values in script at start without scanning QR code by user ?

**Comment by ragavendranbala** [comment]
Yes it's possible, I guess we need to pass the qr detected link to params method. I have successfully created a qr based on my requirements and passed that string info to the save params method
